scripts/fix-comments.py

```diff
@@ -41,7 +41,7 @@
 
 name_map = dict()
 for f in align_files.stdout.splitlines():
-    with open(os.path.join(root_dir, f)) as fh:
+    with open(os.path.join(root_dir, f), encoding="utf-8", newline='\n') as fh:
```

Comment:
```suggestion
    with open(os.path.join(root_dir, f), encoding="utf-8") as fh:
```
This isn't useful for reading .lean files, same below